CRITICAL - 2023-08-16 17:43:50 --> The framework needs the following extension(s) installed and loaded: intl.
in SYSTEMPATH\CodeIgniter.php on line 227.
 1 SYSTEMPATH\CodeIgniter.php(227): CodeIgniter\Exceptions\FrameworkException::forMissingExtension('intl')
 2 SYSTEMPATH\CodeIgniter.php(191): CodeIgniter\CodeIgniter->resolvePlatformExtensions()
 3 FCPATH\index.php(55): CodeIgniter\CodeIgniter->initialize()
ERROR - 2023-08-16 16:28:59 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:28:59 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:29:57 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:29:57 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:30:12 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:30:12 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:34:16 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:34:16 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:36:00 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:36:00 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:36:02 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:36:02 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:39:13 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:39:13 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:39:15 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:39:15 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:39:41 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:39:41 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:39:44 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:39:44 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 16:39:49 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp2\\htdoc...', 173)
#1 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost,8080', 'root', Object(SensitiveParameterValue), 'dismat', 3306, '', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido.  in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp2\htdocs\dismat\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp2\htdocs\dismat\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
#6 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->index()
#7 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-08-16 16:39:49 --> Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for localhost,8080 failed: Host desconocido. 
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `categorias`
WHERE `estado` = :estado:', [...], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Categorias.php(16): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 16:50:49 --> Maximum execution time of 120 seconds exceeded
in SYSTEMPATH\Debug\Exceptions.php on line 170.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-08-16 16:50:52 --> Maximum execution time of 120 seconds exceeded
in SYSTEMPATH\Debug\Exceptions.php on line 170.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2023-08-16 17:25:23 --> Undefined array key "id"
in APPPATH\Views\categorias\categorias.php on line 29.
 1 APPPATH\Views\categorias\categorias.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 29)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 18:11:37 --> Undefined array key "id"
in APPPATH\Views\categorias\categorias.php on line 29.
 1 APPPATH\Views\categorias\categorias.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 29)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 18:12:20 --> Undefined array key "id"
in APPPATH\Views\categorias\categorias.php on line 31.
 1 APPPATH\Views\categorias\categorias.php(31): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 31)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 18:15:08 --> Undefined array key "id"
in APPPATH\Views\categorias\categorias.php on line 35.
 1 APPPATH\Views\categorias\categorias.php(35): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 35)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 19:38:44 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 19:38:44 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:38:44\', \'2023-08-16 19:38:44\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:38:44\', \'2023-08-16 19:38:44\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:38:44\', \'2023-08-16 19:38:44\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 19:42:15 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 19:42:15 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:42:15\', \'2023-08-16 19:42:15\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:42:15\', \'2023-08-16 19:42:15\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:42:15\', \'2023-08-16 19:42:15\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 19:42:46 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 19:42:46 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:42:46\', \'2023-08-16 19:42:46\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:42:46\', \'2023-08-16 19:42:46\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:42:46\', \'2023-08-16 19:42:46\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 19:48:24 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 19:48:24 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:48:24\', \'2023-08-16 19:48:24\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:48:24\', \'2023-08-16 19:48:24\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 19:48:24\', \'2023-08-16 19:48:24\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:02:33 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:02:33 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:02:33\', \'2023-08-16 20:02:33\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:02:33\', \'2023-08-16 20:02:33\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:02:33\', \'2023-08-16 20:02:33\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:03:28 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:03:28 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:03:28\', \'2023-08-16 20:03:28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:03:28\', \'2023-08-16 20:03:28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:03:28\', \'2023-08-16 20:03:28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:06:07 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:06:07 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:06:07\', \'2023-08-16 20:06:07\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:06:07\', \'2023-08-16 20:06:07\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:06:07\', \'2023-08-16 20:06:07\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:10:59 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:10:59 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:10:59\', \'2023-08-16 20:10:59\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:10:59\', \'2023-08-16 20:10:59\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:10:59\', \'2023-08-16 20:10:59\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:11:23 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:11:23 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:11:23\', \'2023-08-16 20:11:23\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:11:23\', \'2023-08-16 20:11:23\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:11:23\', \'2023-08-16 20:11:23\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:23:08 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:23:53 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:24:56 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:24:58 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:25:03 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:27:55 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:28:03 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:28:03 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:28:03\', \'2023-08-16 20:28:03\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:28:03\', \'2023-08-16 20:28:03\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:28:03\', \'2023-08-16 20:28:03\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:29:10 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:34:30 --> Too few arguments to function App\Controllers\Categorias::editar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 42.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:42:52 --> Undefined variable $dato
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dato', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('1')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:42:52 --> Undefined variable $dato
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $dato', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:43:37 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:44:04 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:44:04 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:44:04\', \'2023-08-16 20:44:04\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:44:04\', \'2023-08-16 20:44:04\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:44:04\', \'2023-08-16 20:44:04\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:49:22 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:50:22 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 20:52:58 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:53:05 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:53:05 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:53:05\', \'2023-08-16 20:53:05\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:53:05\', \'2023-08-16 20:53:05\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:53:05\', \'2023-08-16 20:53:05\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:57:32 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-16 20:57:32 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:57:32\', \'2023-08-16 20:57:32\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:57:32\', \'2023-08-16 20:57:32\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:57:32\', \'2023-08-16 20:57:32\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Categorias.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:57:36 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-08-16 20:57:36 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:57:36\', \'2023-08-16 20:57:36\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:57:36\', \'2023-08-16 20:57:36\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:57:36\', \'2023-08-16 20:57:36\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Categorias.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
10 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
11 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 20:59:28 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 20:59:28 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:59:28\', \'2023-08-16 20:59:28\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:59:28\', \'2023-08-16 20:59:28\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 20:59:28\', \'2023-08-16 20:59:28\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:00:40 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:04:17 --> Too few arguments to function App\Controllers\Categorias::eliminar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:06:32 --> Too few arguments to function App\Controllers\Categorias::eliminar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:07:00 --> Too few arguments to function App\Controllers\Categorias::eliminar(), 0 passed in C:\xampp2\htdocs\dismat\system\CodeIgniter.php on line 932 and exactly 1 expected
in APPPATH\Controllers\Categorias.php on line 59.
 1 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar()
 2 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 3 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:07:01 --> Undefined array key "id_categoria"
in APPPATH\Views\categorias\categorias.php on line 29.
 1 APPPATH\Views\categorias\categorias.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_categoria"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 29)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:07:03 --> Undefined array key "id_categoria"
in APPPATH\Views\categorias\categorias.php on line 29.
 1 APPPATH\Views\categorias\categorias.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_categoria"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 29)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:07:20 --> Undefined array key "id_categoria"
in APPPATH\Views\categorias\categorias.php on line 29.
 1 APPPATH\Views\categorias\categorias.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_categoria"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 29)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:07:21 --> Undefined array key "id_categoria"
in APPPATH\Views\categorias\categorias.php on line 29.
 1 APPPATH\Views\categorias\categorias.php(29): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_categoria"', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php', 29)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\categorias.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/categorias', [], true)
 5 APPPATH\Controllers\Categorias.php(20): view('categorias/categorias', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->index()
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:07:40 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Categorias.php(62): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar('1')
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:08:23 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:08:29 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Categorias.php(62): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar('1')
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 21:08:44 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 21:08:44 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:08:44\', \'2023-08-16 21:08:44\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:08:44\', \'2023-08-16 21:08:44\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:08:44\', \'2023-08-16 21:08:44\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:12:11 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:12:18 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Categorias.php(62): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar('1')
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:13:58 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 21:14:59 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 21:14:59 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:14:59\', \'2023-08-16 21:14:59\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:14:59\', \'2023-08-16 21:14:59\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:14:59\', \'2023-08-16 21:14:59\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:21:08 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Categorias.php(62): CodeIgniter\Model->update('6', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar('6')
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:21:14 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:30:31 --> Invalid file: "categorias/eliminadas.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('categorias/eliminadas.php')
 2 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/eliminadas', [], true)
 3 APPPATH\Controllers\Categorias.php(71): view('categorias/eliminadas', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminados()
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:31:59 --> Invalid file: "categorias/eliminadas.php"
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('categorias/eliminadas.php')
 2 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/eliminadas', [], true)
 3 APPPATH\Controllers\Categorias.php(71): view('categorias/eliminadas', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminados()
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-16 21:34:42 --> mysqli_sql_exception: Column 'nombre' cannot be null in C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp2\htdocs\dismat\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `ca...', 0)
#1 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ca...')
#2 C:\xampp2\htdocs\dismat\system\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ca...')
#3 C:\xampp2\htdocs\dismat\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ca...', Array, false)
#4 C:\xampp2\htdocs\dismat\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp2\htdocs\dismat\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp2\htdocs\dismat\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp2\htdocs\dismat\system\BaseModel.php(692): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp2\htdocs\dismat\app\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
#10 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
#11 C:\xampp2\htdocs\dismat\system\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp2\htdocs\dismat\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2023-08-16 21:34:42 --> Column 'nombre' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:34:42\', \'2023-08-16 21:34:42\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(692): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:34:42\', \'2023-08-16 21:34:42\')')
 3 SYSTEMPATH\Database\BaseConnection.php(606): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (NULL, \'2023-08-16 21:34:42\', \'2023-08-16 21:34:42\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `categorias` (`nombre`, `fechaRegistro`, `fechaActualizacion`) VALUES (:nombre:, :fechaRegistro:, :fechaActualizacion:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(692): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Categorias.php(38): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->insertar()
11 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
12 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:39:07 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 924.
 1 SYSTEMPATH\BaseModel.php(924): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(756): CodeIgniter\BaseModel->update([...], [])
 3 APPPATH\Controllers\Categorias.php(62): CodeIgniter\Model->update('1', [...])
 4 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->eliminar('1')
 5 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 6 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:39:24 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:39:33 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-08-16 21:39:40 --> Trying to access array offset on value of type null
in APPPATH\Views\categorias\editar.php on line 7.
 1 APPPATH\Views\categorias\editar.php(7): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to access array offset on value of type null', 'C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php', 7)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp2\\htdocs\\dismat\\app\\Views\\categorias\\editar.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1176): CodeIgniter\View\View->render('categorias/editar', [], true)
 5 APPPATH\Controllers\Categorias.php(49): view('categorias/editar', [...])
 6 SYSTEMPATH\CodeIgniter.php(932): App\Controllers\Categorias->editar('img', 'undraw_profile.svg')
 7 SYSTEMPATH\CodeIgniter.php(497): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categorias))
 8 SYSTEMPATH\CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
